{{cookiecutter.project_slug}}/README.md

```diff
@@ -72,3 +72,62 @@ To create a superuser use the following commands:
 If React frontend was selected during the project creation
 (using our [cookiecutter](https://github.com/sixfeetup/cookiecutter-sixiedjango)), you
 can access it at [http://localhost:3000/](http://localhost:3000/).
+
+## Install Loki for log aggregation and the Kube Prometheus Stack with Grafana Dashboards
+
+### Setup AWS credentials
+
+First export the credentials to your environment variables. Change the values accordingly:
+
+```
+export AWS_ACCESS_KEY_ID='ABC123456'
+export AWS_SECRET_ACCESS_KEY='ABC123456'
+```
+
+Then create a secret in the monitoring namespace:
+
+```
+kubectl create secret generic iam-loki-s3 --from-literal=AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID --from-literal=AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY -n monitoring
+```
+
+### Install monitoring
+
+Now install the loki-stack and kube-prometheus-stack helm charts:
+
+```
+make monitoring-up
+```
+
+If you want to store the logs in an S3 bucket, you will need to include the yaml values `k8s/_monitoring/loki-stack-values.yaml` file for the `helm install loki` command in the Makefile:
+
+```
+helm install loki grafana/loki-stack --values k8s/_monitoring/loki-stack-values.yaml --namespace monitoring --create-namespace
+```
+
+### Connect to Grafana dashboard
+
+Connect to Grafana through local port forwarding, which is the only way to connect currently (since the service Grafana dashboard does not have an ingress entry on it's own)
+
+```
+make monitoring-port-forward
+```
+
+And open http://localhost:8080 on your browser
+
+Login with admin / prom-operator that are the default values. To see these values, run
+
+```
+make monitoring-login
```

Comment:
We should encourage users to change this immediately upon login and store it in a vault.

Reply:
Even better would be a Makefile target to change the password from the default immediately after installion.

Reply:
I added the command to change the password... so I guess this is not needed, right?

Also, we could generate a random password and put that into a YAML file when spinning up loki-stack. Command is still useful, but might be cleaner that way.

Reply:
I agree, let's create a secret with a random password when running make.